docs/README.md

```diff
@@ -64,6 +72,32 @@ If you specify a custom redirectUri you need to define a manifest override as we
 __If you't don specify a custom redirectUri you don't have to add anything to your apps manifest
 file.__
 
+### Automated Auth
+Setup the library for Automated login in which the library reads and present users from
+a json configuration file. This file contains the necessary data to request an access token
+from a third party endpoint. This feature is meant to ease testing.
+
+___Automated login doesn't support token refresh and will default to relogin whenever the token expires.___
+
+```
```

Comment:
You can add kotlin as the language in the code block (see [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)).

Reply:
nice, will do :+1: